v1/branch/branch.gunk

```diff
@@ -233,10 +243,15 @@ type BranchService interface {
 	// +gunk openapiv2.Operation{
 	//         Tags:        []string{"Branch"},
 	//         Description: "Creates a new branch and returns its id.",
-	//         Summary:     "Creates a Branch",
+	//         Summary:     "Create a branch",
 	//         Responses: map[string]openapiv2.Response{
 	//                 "201": openapiv2.Response{
 	//                         Description: "Branch created successfully.",
+	//                         Schema: openapiv2.Schema{
+	//                                 JSONSchema: openapiv2.JSONSchema{
+	//                                         Ref: "#/definitions/branchCreateBranchResponse",
```

Comment:
Should the Ref be "#/definitions/branch`es`CreateBranchResponse?

Reply:
see: https://github.com/openbank/openbank/pull/17#discussion_r314579504